assemblies/prd/pom.xml

```diff
@@ -135,6 +135,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```

Comment:
Remove the version after putting it in the parent pom's main pentaho-karaf-assembly's dependencyManagement section.